app/src/pages/dashboard/components/ANCPriceChart.tsx

```diff
@@ -178,9 +178,10 @@ export class ANCPriceChart extends Component<ANCPriceChartProps> {
             ticks: {
               autoSkip: false,
               maxRotation: this.props.isMobile ? undefined : 0,
-              font: {
-                size: 11,
-              },
+              //@ts-ignore
```

Comment:
For some reason, Chart.js requires a bunch of extra attributes when we pass a function instead of an object.